frinx/common/worker/worker.py

```diff
@@ -224,17 +224,25 @@ def exception_response_handler(self, error: Exception, **kwargs: Any) -> TaskRes
             logs=[TaskExecLog(f'{error_name}: {error}')]
         )
 
-        match error:
-            case ValidationError():
-                formatted_error: DictAny = self._validate_exception_format(error)
-                task_result.logs = [TaskExecLog(f'{error_name}: {formatted_error}')]
-
-                if execution_properties.pass_worker_input_exception_to_task_output:
-                    task_result.output = self._parse_exception_output_path_to_dict(
-                        dot_path={
-                            execution_properties.worker_input_exception_task_output_path: formatted_error
-                        }
-                    )
+        if execution_properties.pass_task_error_to_task_output:
+            match error:
+                case ValidationError():
+                    error_info: str | DictAny = self._validate_exception_format(error)
+                case _:
+                    error_info = str(error)
+            task_result.output = TaskOutput(**{'error': {'error_name': error_name, 'error_info': error_info}})
+        else:
+            match error:
+                case ValidationError():
+                    formatted_error: DictAny = self._validate_exception_format(error)
+                    task_result.logs = [TaskExecLog(f'{error_name}: {formatted_error}')]
+
+                    if execution_properties.pass_worker_input_exception_to_task_output:
+                        task_result.output = self._parse_exception_output_path_to_dict(
+                            dot_path={
+                                execution_properties.worker_input_exception_task_output_path: formatted_error
+                            }
+                        )
```

Comment:
this mean that you can't combine _pass_task_error_to_task_output_ with _worker_input_exception_task_output_path_ if I'm correct. I'm not sure if that is a good

Reply:
If you combine that, the _worker_input_exception_task_output_path_ is skipped in the if-else condition. The new _pass_task_error_to_task_output_ attribute has functionality similar to _worker_input_exception_task_output_path_. It also includes worker input exceptions in the output.

I don't see any real advantage of _worker_input_exception_task_output_path_.